WowPacketParserModule.V7_0_3_22248/Parsers/InstanceHandler.cs

```diff
@@ -74,6 +74,96 @@ public static void HandleCUFProfiles(Packet packet)
             }
         }
 
+        public static void ReadUnkEncouter(Packet packet, params object[] idx)
+        {
+            packet.ReadUInt32("UnkUInt32_13", idx);
+            packet.ReadUInt32("UnkUInt32_14", idx);
+
+            var count1 = packet.ReadInt32("UnkCount1", idx);
+            var count2 = packet.ReadInt32("UnkCount2", idx);
+            var count3 = packet.ReadInt32("UnkCount3", idx);
+
+            for (var j = 0; j < count1; ++j)
+            {
+                packet.ReadInt32("UnkInt32_15", idx, j);
+            }
+
+            for (var j = 0; j < count2; ++j)
+            {
+                packet.ReadInt32("UnkInt32_16", idx, j);
+            }
+
+            for (var j = 0; j < count3; ++j)
+            {
+                ReadUnkEncouter(packet, idx, j);
+            }
+        }
+
+        [Parser(Opcode.SMSG_ENCOUNTER_START)]
+        public static void HandleEncounterStart(Packet packet)
+        {
+            packet.ReadInt32("EncounterID");
+            packet.ReadInt32<DifficultyId>("DifficultyID");
+            packet.ReadInt32("GroupSize");
+
+            // Debug only???
+            var count = packet.ReadInt32("UnkCount");
```

Comment:
Any ideas what's the use of following packet data? never had any packet with it inside

Reply:
same, hadnt find packet with this data